components/datasets/list/DatasetsList.js

```diff
@@ -100,7 +101,9 @@ DatasetsList.defaultProps = {
   },
   getDatasetsFilters: {},
   // Store
-  datasets: []
+  datasets: [],
+  // actions
+  toggleFavourites: () => {}
```

Comment:
I'm not a huge fan of declaring an empty function. I'd rather using isRequired in the Proptypes. But this is a comment